CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Release Notes for autonomX
+
+## Version 0.1.0 - 2020-09-09
+
+This is the first release of autonomX.
+
+New features:
+
+- Run a SNN generator
+- Edit the properties of the SNN generator
+- View the current value of the generator output node
+- View the evolution of the output node of the generator over time
+- See a 2D lattice representation
+- Send the output node of my SNN generator through an OSC output
+- Configure the parameters of the OSC output
+- Receive OSC to the input node of my SNN generator
+- Configure the parameters of the OSC input
+- Have multiple inputs and outputs
+- Draw the inputs and outputs on the 2D lattice representation
+- See different colors for each inputs and outputs
+- Receive OSC from the outside into the inputs of the generator
+- Send the outputs of the generator to the outside with OSC
+- Choose to have the output value be the "mean" of the selected cells
+- See the current value of each input and outputs
+- See the evolution of each input and outputs over the last 10s
+- See the list of all available generators
+- Add a new generator from the list
+- Create a new generator
+- Delete a generator
+- Turn on/off the generators I want to use
```

Comment:
same goes for line 30 `Turn on/off the generators I want to use`; we haven't implemented this yet.